packages/flutter_tools/lib/src/commands/run.dart

```diff
@@ -140,7 +140,15 @@ abstract class RunCommandBase extends FlutterCommand with DeviceBasedDevelopment
               'startup. By default this is main(List<String> args). Specify '
               'this option multiple times each with one argument to pass '
               'multiple arguments to the Dart entrypoint. Currently this is '
-              'only supported on desktop platforms.'
+              'only supported on desktop platforms.',
```

Comment:
This is probably worth adding as a helper method so it can be added to `flutter drive` and possibly `flutter build ios`

Reply:
Done. `DriveCommand` already extends `RunCommandBase`, so no need to call the helper there.